crud.go

```diff
@@ -28,29 +28,29 @@ const (
 	SimulatedVBucketCount = 1024 // Used when hashing doc id -> vbno
 )
 
-// The persistent portion of a lolrus object (the stuff that gets archived to disk.)
-type lolrusData struct {
+// The persistent portion of a Bucket object (the stuff that gets archived to disk.)
+type walrusData struct {
 	LastSeq    uint64                         // Last sequence number assigned
-	Docs       map[string]*lolrusDoc          // Maps doc ID -> lolrusDoc
+	Docs       map[string]*walrusDoc          // Maps doc ID -> walrusDoc
 	DesignDocs map[string]*sgbucket.DesignDoc // Stores source form of design docs
 }
 
 // Simple, inefficient in-memory implementation of Bucket interface.
 // http://ihasabucket.com
-type lolrus struct {
+type Bucket struct {
```

Comment:
I'm not convinced there's a good reason to make walrus.Bucket public - see comment in SG PR.